packages/routes-gen-solid-start/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import path from "path";
+import {exec} from "child_process"
+import {Driver} from "routes-gen/src";
+
+const appDirectory = "./src";
+export const defaultOutputPath: Driver["defaultOutputPath"] = `${appDirectory}/routes.d.ts`;
+export const watchPaths: Driver["watchPaths"] = async () => [
+  `${appDirectory}/routes/**/*.{ts,tsx,js,jsx}`,
+];
+
+type SolidStartRouteData = {
+  id: string,
+  path: string,
+  componentPath: string,
+  dataPath: string
+};
+export const routes: Driver["routes"] = async () => {
+  const scriptPath = path.join(__dirname, 'node_modules', 'solid-start', 'bin.cjs');
+  const parsedRoutes = await new Promise<Array<SolidStartRouteData>>(async (resolve) => {
+    exec(`node ${scriptPath} routes`, (error, stdout, stderr) => {
```

Comment:
WDYT about this?
After I thought of modifying solid-start I notice they are not exposing `{cli}` like `@remix-run/dev`.
It's elegant - so I won't need to create a regex, but it requires node to run which is something I don't like.